src/components/cuda/lcuda_common.h

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file    lcuda_common.h
+ * @author  Anustuv Pal
+ *          anustuv@icl.utk.edu
+ */
+
+#ifndef __LCUDA_COMMON_H__
+#define __LCUDA_COMMON_H__
+
+#include <papi.h>
+
+typedef struct event_record_s {
+    char name[PAPI_2MAX_STR_LEN];
+    char desc[PAPI_2MAX_STR_LEN];
+    int gpu_id;
+    unsigned int evt_code;
+    unsigned int evt_pos;  // index of added event
+    int num_dep;
+    double value;
+    void *info;  // API specific details
```

Comment:
No comment in the same line. Comment in the line before please. We should fix this in other components too.